lib/ts/recipe/session/index.ts

```diff
@@ -160,6 +195,48 @@ export default class SessionWrapper {
             "getOpenIdDiscoveryConfiguration cannot be used without enabling the JWT feature. Please set 'enableJWT: true' when initialising the Session recipe"
         );
     }
+
+    static fetchAndSetClaim(sessionHandle: string, claim: SessionClaim<any>, userContext: any = {}): Promise<void> {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.fetchAndSetClaim({
+            sessionHandle,
+            claim,
+            userContext,
+        });
+    }
+
+    static setClaimValue<T>(
+        sessionHandle: string,
+        claim: SessionClaim<T>,
+        value: T,
+        userContext: any = {}
+    ): Promise<void> {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.setClaimValue({
+            sessionHandle,
+            claim,
+            value,
+            userContext,
+        });
+    }
+
+    static getClaimValue<T>(
+        sessionHandle: string,
+        claim: SessionClaim<T>,
+        userContext: any = {}
+    ): Promise<T | undefined> {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getClaimValue({
+            sessionHandle,
+            claim,
+            userContext,
+        });
+    }
+
+    static removeClaim(sessionHandle: string, claim: SessionClaim<any>, userContext: any = {}): Promise<void> {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.removeClaim({
+            sessionHandle,
+            claim,
+            userContext,
+        });
+    }
```

Comment:
We should also expose a function here to validate (not assert) a claim based on:
- session handle as the input
- JWT as the input -> in the context of a microservice auth (to discuss)

Reply:
Let's discuss these next week.

Reply:
So we can have a function like this:
```
validateClaimsForSessionHandle(sessionHandle: string, overrideGlobalClaimValidators, userContext?) => Promise<ClaimValidationError[]> ( we can update the access token of the other session as well)

validateClaimsInJWTPayload(jwtPayload: JSONObject, overrideGlobalClaimValidators, userContext?) => Promise<ClaimValidationError[]> (just call validate without calling shouldRefetch, since we cannot update this anyway)
```